e2e/internal/e2e/singularitycmd.go

```diff
@@ -262,6 +262,7 @@ type SingularityCmdOp func(*singularityCmd)
 
 // singularityCmd defines a Singularity command execution test.
 type singularityCmd struct {
+	globOptions []string
```

Comment:
I'd prefer `globalOptions` even if it's longer.... because `glob` does mean something else.